src/providers/local/staking/userUserStakingData.ts

```diff
@@ -307,7 +264,7 @@ export default function useUserStakingData(
     () => isWalletReady.value && userGaugeShares.value.length > 0 && !isL2.value
   );
 
-  const { data: poolBoosts, isLoading: isLoadingBoosts } = useQuery<
+  const { data: poolBoosts, isInitialLoading: isLoadingBoosts } = useQuery<
```

Comment:
Have to be careful with queries that are not always enabled like this one here. Their `isLoading` state stays `false` until they are enabled. This caused the `StakePreview.vue` to show perpetual loading block.

<img width="1070" alt="Screenshot 2023-01-05 at 14 55 35" src="https://user-images.githubusercontent.com/28311328/210785321-f09977b4-738b-4c73-8728-25e90de89475.png">

Checking the `isInitialLoading` instead works here. https://tanstack.com/query/v4/docs/react/guides/disabling-queries

Reply:
I didn't catch that braking change in the docs. I think that we will have more cases like this one.